src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -22,17 +22,30 @@
 #include <ripple/app/main/Application.h>
 #include <ripple/app/misc/Validations.h>
 #include <ripple/app/data/DatabaseCon.h>
+#include <ripple/core/ConfigSections.h>
 #include <boost/format.hpp>
+#include <boost/tokenizer.hpp>
 
 namespace ripple {
 
+namespace detail
+{
+/** Amendments that are "compiled into" this app
+
+    Add amendments to this collection at build time to enable them on this
+   server. The first element of the pair is the id, the second element is
+   the "friendly name".
+*/
+std::set<std::pair<std::string, std::string>> PreEnabledAmendments;
+}
+
 /** Track the list of "amendments"
 
     An "amendment" is an option that can affect transaction processing
```

Comment:
> Other places in rippled are assuming an untagged uint256

When I first showed up at Ripple, we had *no* tagged uints.  Getting rid of
those was an early assignment of mine - I did all the uint160s but didn't
finish the uint256s because I got diverted to something else, and because
at the time we only had one uint256 type.

It depends really on how much work you think it is.  But I find it much
easier to read an actual type than a generic uint256.

On Tue, Jan 13, 2015 at 6:33 PM, seelabs <notifications@github.com> wrote:

> In src/ripple/app/misc/AmendmentTableImpl.cpp
> <https://github.com/ripple/rippled/pull/795#discussion-diff-22905240>:
>
> >
> >  namespace ripple {
> >
> > +namespace detail
> > +{
> > +/** Amendments that are "compiled into" this app
> > +
> > +    Add amendments to this collection at build time to enable them on this
> > +   server. The first element of the pair is the id, the second element is
> > +   the "friendly name".
> > +*/
> > +std::set<std::pair<std::string, std::string>> PreEnabledAmendments;
> > +}
> > +
> >  /** Track the list of "amendments"
> >
> >      An "amendment" is an option that can affect transaction processing
>
> I made this change to see what it effects and my opinion is it should be a
> separate ticket. Other places in rippled are assuming an untagged uint256
> and making this change seem too unrelated to the original ticket.
>
> I don't think the change is difficult and am happy to make it if you fell
> strongly we should make it with this ticket.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ripple/rippled/pull/795/files#r22905240>.
>